frontend/src/components/RisModForm.vue

```diff
@@ -161,11 +165,27 @@ function modTypeLabel(modType: ModType | "") {
         variant="tertiary"
         @click.prevent="$emit('generate-preview')"
       />
-      <RisTextButton
-        label="Speichern"
-        :icon="CheckIcon"
-        @click.prevent="$emit('update-mod')"
-      />
+
+      <div class="relative">
+        <RisTooltip
+          v-slot="{ ariaDescribedby }"
+          :visible="isUpdatingFinished"
+          :title="
+            updateError ? 'Fehler beim Speichern' : 'Speichern erfolgreich'
+          "
+          alignment="right"
+          attachment="top"
+          :variant="updateError ? 'error' : 'success'"
```

Comment:
Question: Is the "show success message" a general pattern we're using? Even though it's possible with the tooltip I haven't heard about this from design. But might've missed it :)

Reply:
We have a design for a success message in figma. But now that you mention it it's not actually using the tooltip and I'm unsure if it is applicable here 🤔

Reply:
I would keep it for now anyway as it is (in my opinion) still better to have some feedback that it worked than none at all

Reply:
Fine with me, but could you check with design about this and ask if this is something we should use everywhere? Thanks!